interpreter-manager/interpreterManager.h

```diff
@@ -0,0 +1,23 @@
+/*
+This file defines the class for the interpreter manager class.
+*/
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+
+//Micropython Modules are in C
+extern "C" {
+    #pragma once
+    #include "py/compile.h"
+    #include "py/runtime.h"
+    #include "py/gc.h"
+    #include "py/stackctrl.h"
```

Comment:
I think we don't need `runtime` & `compile` headers. Can we remove them if we do not use it here?